R/proportional_hazards_data.R

```diff
@@ -16,24 +16,6 @@ make_proportional_hazards_survival <- function() {
   parsnip::set_dependency("proportional_hazards", eng = "survival", pkg = "riskRegression")
```

Comment:
I don't think that we need `riskRegression` anymore.

Reply:
thanks, I've removed it